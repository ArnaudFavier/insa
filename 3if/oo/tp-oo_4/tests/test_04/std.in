S segment1 0 0 4 0
S segment2 -2 -2 2 2
S segment3 -10 -10 -6 -10
LIST
MOVE segment3 10 10
LIST
HIT segment1 0 0
HIT segment1 4 0
HIT segment1 2 0
HIT segment1 5 0
HIT segment1 -1 0
HIT segment1 2 1
HIT segment2 0 0
HIT segment2 -2 -2
HIT segment2 -2 -1
HIT segment3 0 0
HIT segment3 4 0
HIT segment3 2 0
HIT segment3 5 0
HIT segment3 -1 0
HIT segment3 2 1
EXIT
