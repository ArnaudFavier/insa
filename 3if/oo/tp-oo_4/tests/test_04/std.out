OK
OK
OK
segment1 S [(0;0) (4;0)]
segment2 S [(-2;-2) (2;2)]
segment3 S [(-10;-10) (-6;-10)]
OK
segment1 S [(0;0) (4;0)]
segment2 S [(-2;-2) (2;2)]
segment3 S [(0;0) (4;0)]
YES
YES
YES
NO
NO
NO
YES
YES
NO
YES
YES
YES
NO
NO
NO
